docs/docs/modules/model_io/chat/function_calling.mdx

```diff
@@ -1,68 +1,117 @@
 ---
 sidebar_position: 2
-title: Function calling
+title: Tool calling
 ---
 
-# Function calling
-
-A growing number of chat models, like
-[OpenAI](https://platform.openai.com/docs/guides/function-calling),
-[Gemini](https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/function-calling),
-etc., have a function-calling API that lets you describe functions and
-their arguments, and have the model return a JSON object with a function
-to invoke and the inputs to that function. Function-calling is extremely
-useful for building [tool-using chains and
-agents](/docs/use_cases/tool_use/), and for getting
-structured outputs from models more generally.
-
-LangChain comes with a number of utilities to make function-calling
-easy. Namely, it comes with:
-
-- simple syntax for binding functions to models
-- converters for formatting various types of objects to the expected
-  function schemas
-- output parsers for extracting the function invocations from API
-  responses
-- chains for getting structured outputs from a model, built on top of
-  function calling
-
-We’ll focus here on the first two points. For a detailed guide on output
-parsing check out the [OpenAI Tools output
-parsers](/docs/modules/model_io/output_parsers/types/openai_tools/)
-and to see the structured output chains check out the [Structured output
-guide](/docs/modules/model_io/chat/structured_output/).
-
-Before getting started make sure you have `langchain-core` installed.
+# Tool calling
 
-```python
-%pip install -qU langchain-core langchain-openai
+# Calling Tools
+
+Tool calling is a common feature of LLM applications. A [tool call](https://api.python.langchain.com/en/latest/messages/langchain_core.messages.tool.ToolCall.html#langchain_core.messages.tool.ToolCall) 
+represents a call to a specific tool, and includes a name, arguments dict, and 
+(optionally) an identifier. The arguments dict is structured 
+`{argument_name: argument_value}`.
+
+Many LLM providers, including Anthropic, Cohere, Google, Mistral, OpenAI, and others, 
```

Comment:
Make these links

Reply:
thanks.

for google link I almost linked to google.com